airflow/contrib/hooks/gcp_spanner_hook.py

```diff
@@ -147,15 +149,13 @@ def _apply_to_instance(self, project_id, instance_id, configuration_name, node_c
         :param func: Method of the instance to be called.
         :type func: Callable
         """
-        client = self.get_client(project_id)
-        instance = client.instance(instance_id,
-                                   configuration_name=configuration_name,
-                                   node_count=node_count,
-                                   display_name=display_name)
+        instance = self.get_client(project_id).instance(
```

Comment:
I was going to ask for tests of changes to the hook, but I see that we don't test this hook at all at the moment. Do you have time to add tests in this PR, or should we add them in a separate PR?